packages/dcos-integration-test/extra/test_networking.py

```diff
@@ -205,44 +354,52 @@ def reduce_logging():
 
 @pytest.mark.skipif(not lb_enabled(), reason='Load Balancer disabled')
 def test_vip(dcos_api_session, reduce_logging):
-    '''Test VIPs between the following source and destination configurations:
-        * containers: DOCKER, UCR and NONE
-        * networks: USER, BRIDGE (docker only), HOST
-        * agents: source and destnations on same agent or different agents
-        * vips: named and unnamed vip
-
+    '''Test every permutation of VIP
     '''
-    addrs = [['1.1.1.{}:{}', '1.1.1.{}:{}'],
-             ['/namedvip{}:{}', 'namedvip{}.marathon.l4lb.thisdcos.directory:{}']]
     # tests
     # UCR doesn't support BRIDGE mode
-    permutations = [[c, vi, va, sh, vn, pn]
+    permutations = [[c, ss, sh, vn, pn]
```

Comment:
i guess i am not sure what the recommended naming style is in python.  Just typically you want to use single letter names for iterators in loops that are scoped only within the loop.  so its easy to identify which variables are scoped locally to the loop and which ones are scoped to the entire function.

@orsenthil @mellenburg any opinions on the matter?